tanner/create_tables.py

```diff
@@ -0,0 +1,71 @@
+import asyncio
+
+
+class CreateTables:
+    @staticmethod
+    async def create_data_tables(pg_client):
+        """Create all the required tables in
+            the postgres database
+
+        Arguments:
+            pg_client {aiopg.pool.Pool}
+        """
+        async with pg_client.acquire() as conn:
+            async with conn.cursor() as cur:
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "paths" (
+                    "sess_uuid" UUID PRIMARY KEY NOT NULL UNIQUE,
+                    "path" TEXT NOT NULL,
+                    "timestamp" TIMESTAMP NOT NULL,
+                    "response_status" INT NOT NULL,
+                    "attack_type" INT NOT NULL
+                )
+                """
+                )
+
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "cookies" (
+                    "sess_uuid" UUID PRIMARY KEY NOT NULL UNIQUE,
+                    "key" TEXT NULL,
+                    "value" TEXT NULL
+                )
+                """
+                )
+
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "session_data" (
+                    "sess_uuid" UUID NOT NULL PRIMARY KEY,
+                    "snare_uuid" TEXT NOT NULL,
+                    "peer.ip" INET NOT NULL,
+                    "peer.port" INT NOT NULL,
+                    "location.country" TEXT NULL,
+                    "location.country_code" TEXT NULL,
+                    "location.city" TEXT NULL,
+                    "location.zip_code" INT NULL,
+                    "user_agent" TEXT NOT NULL,
+                    "start_time" TIMESTAMP,
+                    "end_time" TIMESTAMP,
+                    "requests_in_second" FLOAT NULL,
```

Comment:
```suggestion
                    "requests_per_second" FLOAT NULL,
```
you could also abbreviate it to `rps` and add a comment explaining the field: `comment on column rps is 'requests per second';`